app/src/modules/autoUpdater.js

```diff
@@ -4,8 +4,8 @@ import {
   IPC_DOWNLOAD_UPDATE_PROGRESS,
   IPC_DOWNLOAD_UPDATE_START,
```

Comment:
Strangeness but unrelated to changes here (since they also happen in source branch)

- trying to connect a URI with 2 dApps, gives correct error message. But if trying to enter with the same uri again gives another error message (invalid connection URI)
- Rejecting a sign_transaction in the desktop app doesnt close the modal in wallet connect app (feels like it should?)
- Second transaction sign doesnt bring up the password modal
- Even if i only connect account2 to wallet connect, i still can sign being on account1, and it show successfully signed